rgd/geodata/models/common.py

```diff
@@ -206,26 +206,42 @@ def yield_local_path(self, vsi=False):
         ----------
         vsi : bool
             If FUSE fails, fallback to a Virtual File Systems URL. See
-            ``get_local_vsi_path``. This is especially useful if the file
+            ``get_vsi_path``. This is especially useful if the file
             is being utilized by GDAL and FUSE is not set up.
 
         """
         if self.type == FileSourceType.URL and precheck_fuse(self.get_url()):
-            return url_file_to_fuse_path(self.get_url())
+            return url_file_to_fuse_path(self.get_url(internal=True))
         elif vsi:
             logger.info('`yield_local_path` falling back to Virtual File System URL.')
-            return self.yield_local_vsi_path()
+            return self.yield_vsi_path(internal=True)
```

Comment:
Hardsetting `internal=True` here because `yield_local_path()` should only ever be used in a local context for ETL routines or other server-side operations